DateTime.js

```diff
@@ -133,10 +136,12 @@ var Datetime = createClass({
 			formats.time = locale.longDateFormat('LT');
 		}
 
-		formats.datetime = formats.date && formats.time ?
-			formats.date + ' ' + formats.time :
-			formats.date || formats.time
-		;
+                if ( !formats.datetime  || formats.datetime === true ) {
```

Comment:
I find this to be weird, we are checking for either `falsey` (which it is by default) or `true`. When would this entire expression not be `true`?